# This message type includes the general
# KPI parameters.
string current_region_name

float64 kpi1    # Distance travelled
float64 kpi2    # Idle time
float64 kpi3    # Motion time
float64 kpi4    # Efficiency

float64 reg_1_kpi5
float64 reg_1_kpi6
float64 reg_1_kpi7
float64 reg_1_kpi8
float64 reg_1_kpi9

float64 reg_2_kpi5
float64 reg_2_kpi6
float64 reg_2_kpi7
float64 reg_2_kpi8
float64 reg_2_kpi9

float64 reg_3_kpi5
float64 reg_3_kpi6
float64 reg_3_kpi7
float64 reg_3_kpi8
float64 reg_3_kpi9

float64 reg_4_kpi5
float64 reg_4_kpi6
float64 reg_4_kpi7
float64 reg_4_kpi8
float64 reg_4_kpi9